docs/source/markdown/options/userns.container.md

```diff
@@ -4,16 +4,16 @@
 ####> are applicable to all of those.
 #### **--userns**=*mode*
 
-Set the user namespace mode for the container. It defaults to the **PODMAN_USERNS** environment variable. An empty value ("") means user namespaces are disabled unless an explicit mapping is set with the **--uidmap** and **--gidmap** options.
+Set the user namespace mode for the container. It defaults to the **PODMAN_USERNS** environment variable. An empty value ("") or **host** means user namespaces are disabled, unless an explicit mapping is set with the **--uidmap** and **--gidmap** options or podman is rootless.
```

Comment:
I'm not at all gronking "or **host**".  Does that mean `--userns=127.0.0.1` and `--userns=myhost.mycompany.com` are valid?  There is no mention of host prior to this, and only one way down lower in this page.  Examples for this, and other possible settings that might be found in this file would be a very welcomed addition to this man page.

Reply:
> An empty value ("") or **host** means …

This wording is chosen in order to emphasize that empty value and **host** have the same meaning.  I am not aware of open tickets, claiming that it is unclear how to interpret **host**, except https://github.com/containers/podman/issues/15764, which is from me.

Reply:
I think `host` is fine, that has the same meaning as any other namespace